ignite/handlers/lr_finder.py

```diff
@@ -137,6 +137,14 @@ def _reset(self, trainer: Engine) -> None:
     def _log_lr_and_loss(self, trainer: Engine, output_transform: Callable, smooth_f: float, diverge_th: float) -> None:
         output = trainer.state.output
         loss = output_transform(output)
+        if not isinstance(loss, float):
+            if isinstance(loss, torch.Tensor):
```

Comment:
We have to check that tensor is 0-d or 1-d with one-element before applying `.item()`.

Reply:
I missed that, thanks for pointing that out!